.github/workflows/stale.yml

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# https://github.com/actions/stale
+
+name: Stale
+
+on:
+  workflow_call:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  waiting-for-feedback:
+    name: 'Manage issues, PRs with waiting-for-feedback label'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/stale@5bef64f19d7facfb25b37b414482c7164d639639 # v9
+        with:
+          days-before-stale: '28'
+          days-before-close: '7'
+          stale-issue-message: 'This issue is stale because it is waiting for feedback until 4 weeks. Remove stale label or comment or this will be closed in 7 days.'
+          close-issue-message: 'This issue has been closed because not received a response for 4 weeks.'
+          stale-pr-message: 'This pull request is stale because it is waiting for feedback until 4 weeks. Remove stale label or comment or this will be closed in 7 days.'
```

Comment:
it has been waiting for feedback for 4 weeks. Remove the stale label or comment on this PR, or it will be automatically closed in 7 days.